### PR TITLE
Feat/2526 build a static program entity page(update)

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
# Description of changes

Alexis requested two updates: a feature flag & add links to program-entity page from the file repo table.

See her comments and screenshots here:
https://github.com/icgc-argo/platform-ui/issues/2526#issuecomment-1434838276

## Type of Change

- [ ] Bug
- [ ] Dependency updates
- [x] Feature
- [ ] Refactoring
- [ ] Release candidate
- [ ] Styling
- [ ] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [x] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [x] No back-end changes
- [x] Manually tested changes
- [ ] Added copyrights to new files
- [x] Connected ticket to PR

Feature flag added. Turn true or false in .env file:
Screenshot 2023-02-17 at 4 39 26 PM

File Repo table links are available when feature flag is enabled(top) and could be turn off(bottom):
Screenshot 2023-02-17 at 4 38 55 PM
Screenshot 2023-02-17 at 4 40 05 PM

Program Id is dynamic on Program Entity Page:
Screenshot 2023-02-17 at 4 38 34 PM

### CODE REVIEW

File: pages/program/index.tsx [hunk 1]
```diff
@@ -19,9 +19,21 @@
```
Comment (demariadaniel) [inline pages/program/index.tsx:19]: I think we can delete this file correct because we have `[programId].tsx` now?

Comment (daniel-cy-lu) [reply pages/program/index.tsx:19]: oh yes. I thought it has been deleted. It probably was the auto merge conflict resolution. I will remove it!

**Review by samrichca — APPROVED** [review]
looks great! 💯 LGTM

**Comment by daniel-cy-lu** [comment]
> looks great! 💯 LGTM

Thanks @samrichca @demariadaniel

**Pull request merged**